### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by joramot**
**### Project requirements**

- Set up a new React application.
- Create a directory for the reusable components:.
- Install React Router
- Set up the router inside of App.
- Add 2 routes in the application, each should render a page component:
    >  /
    >  /categories

- Create components inside of the /components directory:
    >  Books.
    >  Categories.

- Create a form to add new books
- Add navigation to link to the 2 routes.

### CODE REVIEW

**Review by algerina — COMMENTED** [review]
Hi @joramot , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project without setting up Linters. Kindly setup Linters correctly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      <div className="">
+        { books.map((book) => (
+          <p key={book.id}>
+            {book.title}
+            {' '}
+            written by
+            {' '}
+            {book.author}
+          </p>
+        )) }
```
Comment (codepantha) [inline src/components/Books.js]: - [ ] Please create a `<Book />` component that takes the book data as props and renders the title and the author.
- [ ] Please ensure to add a button to delete the book. Kindly note that the button wouldn't remove the books yet; it is just a placeholder for now 👍

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <hr />
+      <div className="">
+        <h2>Add new book</h2>
+        <form>
+          <input type="text" placeholder="Book title" />
+          <select name="catagory" id="category">
+            <option value="NodeJS">NodeJS</option>
+            <option value="React">React</option>
+            <option value="Javascript">Javascript</option>
+          </select>
```
Comment (codepantha) [inline src/components/Books.js]: - [ ] According to the project requirements, please replace the select input for "category" with an input for "author"

image
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @joramot 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Set up project with components 📌 
✔️ Implemented routing with `react-router-dom` 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (DeliceLydia) [reply src/components/Books.js]: Addressed

File: src/components/Books.js (hunk 2 above)
Comment (DeliceLydia) [reply src/components/Books.js]: Addressed

**Review by DeliceLydia — APPROVED** [review]
## Hi @joramot, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Addressed changes requested

### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**